Tools/gen_manifest

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
```

Comment:
AFAIK you should specify Python version here for better compatibility with various Linux distributions (i.e. `#!/usr/bin/env python2`).

Reply:
DONE.